assembl/static2/js/app/components/administration/survey/questionsForm.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { gql, withApollo } from 'react-apollo';
+import { FormGroup } from 'react-bootstrap';
+
+import QuestionTitle from './questionTitle';
+
+const GetThematics = gql`
+{
+  thematics(identifier:"survey") {
+    id,
+    titleEntries {
+      localeCode,
+      value
+    },
+    imgUrl,
+    video {
+      title,
+      description,
+      htmlCode
+    }
+    questions {
+      titleEntries {
+        localeCode,
+        value
+      }
+    }
+  }
+}
+`;
+
+const QuestionsForm = ({ client, thematicId, lang }) => {
+  const thematicsData = client.readQuery({ query: GetThematics });
+
+  const thematics = thematicsData.thematics;
+  const thematic = thematics.find((t) => {
+    return String(t.id) === thematicId;
+  });
+
+  const thematicIndex = thematics.findIndex((t) => {
+    return String(t.id) === thematicId;
+  });
+
+  const questions = thematic ? thematic.questions : [];
+
+  const addQuestion = () => {
+    thematicsData.thematics[thematicIndex].questions.push({
```

Comment:
in the worst case scenario, `findIndex` returns `-1` (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex?v=control)
I fix this